### PR TITLE
Improved testing

### PR DESCRIPTION
**Pull request by thedrow**
Added 2.3 to the build matrix and added property based testing using rantly.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,16 @@
 language: ruby
+sudo: false
+cache:
+  bundler: true
+  directories:
+    - $HOME/.ccache
+
 rvm:
   - 2.2.0
+  - 2.3.2
```
Comment (miyucy) [inline .travis.yml]: cannot use 2.3.3? (i'm not familar to travis)

**Comment by miyucy** [comment]
thanks so much!

File: spec/brotli_spec.rb [hunk 2]
```diff
@@ -41,19 +57,28 @@
...
+
+      compressed = Brotli.deflate(data)
+      brotli_compressed_bytesize = compressed.bytesize
+      brotli_compression_ratio = (compressed.bytesize / data.bytesize.to_f * 100).round(3)
+      puts "Brotli size: #{brotli_compressed_bytesize}, compress ratio: #{brotli_compression_ratio} %"
 
-    compressed = Brotli.deflate(data)
-    puts "Brotli size: #{compressed.bytesize}, compress ratio: #{(compressed.bytesize / data.bytesize.to_f * 100).round(3)} %"
+      expect(brotli_compressed_bytesize).to be < zlib_compressed_bytesize
+      expect(brotli_compression_ratio).to be < zlib_compression_ratio
```
Comment (miyucy) [inline spec/brotli_spec.rb]: 💪

File: .travis.yml (hunk 1 above)
Comment (thedrow) [inline .travis.yml]: Oh right. I forgot it was released.

**Pull request merged**